pkgs/development/compilers/ghcjs-ng/8.6/dep-overrides.nix

```diff
@@ -1,7 +1,11 @@
-{ haskellLib }:
+{ haskellLib, happy_1_19_5 }:
```

Comment:
Shouldn't you just use `self.happy_1_19_5` rather than passing it in this way? Why is this version needed anyway?

Reply:
ghc-api-ghcjs has the constraint `happy >= 1.19 && <= 1.19.9`.

Reply:
I didn't get it from `self` because `common-overrides.nix` didn't use `self`.